### PR TITLE
chore: update dependencies

### PR DESCRIPTION
**Pull request by Yelinz**
Update ember source to 3.22

### CODE REVIEW

File: .dependabot/config.yml [hunk 1]
```diff
@@ -8,9 +8,6 @@ update_configs:
...
           update_type: "semver:minor"
-    ignored_updates:
-      - match:
-          # Can not be updated to the latest version because it requires ember 3.11+
-          # but this addon supports the last 2 LTS vesions i.e. 3.8 & 3.12
-          dependency_name: "ember-power-select"
-          version_requirement: ">=3"
+    commit_message:
+      prefix: "chore"
+      include_scope: true
```
Comment (anehx) [inline .dependabot/config.yml:13]: Isn't this the default?

**Review by anehx — CHANGES_REQUESTED** [review]
Looks good, however please mark it as `chore!` and as breaking change correctly: `BREAKING CHANGE:` at the beginning of the commit message. Also, state clearly what the breaking change is e.g. "This drops support for Ember LTS 3.12"

https://www.conventionalcommits.org/en/v1.0.0/#commit-message-with-both--and-breaking-change-footer

File: .dependabot/config.yml (hunk 1 above)
Comment (Yelinz) [reply .dependabot/config.yml:13]: No it isnt, if you look at the current dependabot commits they arent prefixed

**Comment by Yelinz** [comment]
> Looks good, however please mark it as `chore!` and as breaking change correctly: `BREAKING CHANGE:` at the beginning of the commit message. Also, state clearly what the breaking change is e.g. "This drops support for Ember LTS 3.12"
> 
> https://www.conventionalcommits.org/en/v1.0.0/#commit-message-with-both--and-breaking-change-footer

I tried using chore! but the commit linter said it was invalid

**Comment by Yelinz** [comment]
Seems like we have to update out semantic release package to use newer versions of commitlint for the chore! style

File: .dependabot/config.yml (hunk 1 above)
Comment (anehx) [reply .dependabot/config.yml:13]: Ah I see, we didn't use conventional commits long enough for dependabot to detect it. In that case, please use this:

```yml
    commit_message:
      prefix: "fix"
      prefix_development: "chore"
      include_scope: true
```

Comment (Yelinz) [reply .dependabot/config.yml:14]: Ok changed

**Comment by Yelinz** [comment]
@anehx Updated our semantic release config and now the ! syntax in the commit works

**Review by anehx — APPROVED** [review]

**Pull request merged**